modules/preview-react/menu/stories/examples/ManyItems.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import {Menu, MenuItem} from '@workday/canvas-kit-preview-react/menu';
+
+export const ManyItems = () => {
+  return (
+    <Menu title="Menu Titles">
+      {'One Two Three Four Five Six Seven Eight Nine Ten Eleven Twelve Thirteen Fourteen Fifteen Sixteen Seventeen'
```

Comment:
Our [Menu design docs](https://design.workday.com/components/popups/menus) mention that menus should have 7-15 items. I'm not sure where the research came from on that. But it feels like we can align here and keep the example to 15. 

```suggestion
      {'One Two Three Four Five Six Seven Eight Nine Ten Eleven Twelve Thirteen Fourteen Fifteen'
```